community/planning/dlt_polling_monitor.md

```diff
@@ -67,6 +70,34 @@ The DLT Polling Monitor will work through the queue in the following manner:
    4. Updates db with `update_batch_statuses`
 4. Reruns on the specified `poll_interval`
 
+## Interaction with other services
+
+<div class="mermaid">
+sequenceDiagram
+    participant Poller as DLT Polling Monitor
+    participant EventMonitor as DLT Event Monitor
+    participant Gridd as Grid Daemon
+    participant Ledger as Splinter / Sawtooth
+    participant DB as Grid DB
+    Gridd->>Poller: Standup DLT Polling Monitor
+    loop Poll
+        Poller->>+DB: Select all pending batches
+        DB-->>-Poller: 
+        Note right of Poller: Scabbard: Group batches by service
+        Note right of Poller: Sawtooth: Group all batches together
+        loop For each group
+            Poller->>+Ledger: GET /batch_statuses
+            Ledger-->>-Poller: 
+        end
+        Poller->>+DB: Update statuses
+    end
+    Gridd->>EventMonitor: Standup DLT Event Monitor
+    EventMonitor->>+Ledger: Connect to websocket DLT event stream
+    loop Every event
+        EventMonitor->>+Poller: Trigger poll
+    end
+</div>
+
 ## Public Traits and Structs
```

Comment:
I think this sequence diagram looks v good, but maybe we could just add a little sentence to give some clarification for what this diagram is for ? Beyond the title, that is, perhaps would just give more context for that diagram. As it's not super clear what we mean here by "other services", but just pointing out that we mean the DLTs and whatnot

Reply:
Added!